docs/public-networks/reference/cli/options.md

```diff
@@ -2636,16 +2636,17 @@ network="holesky"
 
 The predefined network configuration. The default is `mainnet`.
 
-Possible values are:
-
-| Network   | Chain | Type        | Default Sync Mode  | Description                                                    |
-| :-------- | :---- | :-----------| :----------------- | :------------------------------------------------------------- |
-| `mainnet` | ETH   | Production  | [FAST](#sync-mode) | The main network                                               |
-| `holesky` | ETH   | Test        | [FAST](#sync-mode) | A PoS network                                                  |
-| `sepolia` | ETH   | Test        | [FAST](#sync-mode) | A PoS network                                                  |
-| `dev`     | ETH   | Development | [FULL](#sync-mode) | A PoW network with a low difficulty to enable local CPU mining |
-| `classic` | ETC   | Production  | [FAST](#sync-mode) | The main Ethereum Classic network                              |
-| `mordor ` | ETC   | Test        | [FAST](#sync-mode) | A PoW network                                                  |
+Possible values include the following:
+
+| Network   | Chain | Type        | Default Sync Mode  | Consensus Mechanism      | Description                                                                          |
+| :-------- | :---- | :-----------| :----------------- | :----------------------- | :----------------------------------------------------------------------------------- |
+| `mainnet` | ETH   | Production  | [FAST](#sync-mode) | A PoS network            | The main [Ethereum network](https://ethereum.org/en/developers/docs/networks/)       |
+| `holesky` | ETH   | Test        | [FAST](#sync-mode) | A PoS network            | Multi-client testnet [Hoelsky](https://holesky.dev)                                  |
+| `sepolia` | ETH   | Test        | [FAST](#sync-mode) | A PoS network            | Multi-client testnet [Sepolia](https://sepolia.dev)                                  |
+| `lukso`   | ETH   | Production  | [FAST](#sync-mode) | A PoS network            | Network for the [Lukso chain](https://lukso.network/)                                |
```

Comment:
```suggestion
| `lukso`   | ETH   | Test  | [FAST](#sync-mode) | A PoS network            | Network for the [Lukso chain](https://lukso.network/)                                |
```

Reply:
@alexandratran Sorry this is production according to the [Teku docs](https://docs.teku.consensys.io/reference/cli#network)